classes/plugins/PluginGalleryDAO.inc.php

```diff
@@ -56,7 +56,9 @@ private function _getDocument() {
 		$client = $application->getHttpClient();
 		$versionDao = DAORegistry::getDAO('VersionDAO');
 		$currentVersion = $versionDao->getCurrentVersion();
-		$response = $client->request('GET', PLUGIN_GALLERY_XML_URL, ['query' => ['application' => $application->getName(), 'version' => $currentVersion->getVersionString()]]);
+		$pluginGalleryUrl = PLUGIN_GALLERY_XML_URL;
+		\HookRegistry::call('PluginGallery::getPluginGalleryUrl', array(&$pluginGalleryUrl));
```

Comment:
```suggestion
		HookRegistry::call('PluginGallery::getPluginGalleryUrl', array(&$pluginGalleryUrl));
```
OJS 3.3.0-x doesn't use namespaces, so the backslash isn't necessary.